From e4bfcab810c8b96b9d18b5f3c39e8e97db055d06 Mon Sep 17 00:00:00 2001
From: zhenpengzheng <zhenpengzheng@net-swift.com>
Date: Tue, 13 Jul 2021 12:01:54 +0800
Subject: [PATCH] x86/config: Set CONFIG_TXGBE=m by default
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: e4bfcab810c8b96b9d18b5f3c39e8e97db055d06


driver inclusion
category: feature
bugzilla: 50777
CVE: NA

-------------------------------------------------------------------------

Ensure the netswift 10G NIC driver ko can be distributed in ISO on X86.

Signed-off-by: zhenpengzheng <zhenpengzheng@net-swift.com>
Signed-off-by: Zhen Lei <thunder.leizhen@huawei.com>
Acked-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/x86/configs/openeuler_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index b67756dc1db7..aebbf224c870 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -2674,6 +2674,8 @@ CONFIG_NET_VENDOR_GOOGLE=y
 # CONFIG_GVE is not set
 CONFIG_NET_VENDOR_HUAWEI=y
 CONFIG_HINIC=m
+CONFIG_NET_VENDOR_NETSWIFT=y
+CONFIG_TXGBE=m
 # CONFIG_NET_VENDOR_I825XX is not set
 CONFIG_NET_VENDOR_INTEL=y
 # CONFIG_E100 is not set
-- 
2.26.2

